docker/k8s-sglang-service.yaml

```diff
@@ -35,10 +35,20 @@ spec:
           env:
             - name: HF_TOKEN
               value: <secret>
+            - name: NUMBA_CACHE_DIR
+              value: /tmp/.cache/numba
+            - name: OUTLINES_CACHE_DIR
+              value: /tmp/.cache/outlines
+            - name: TRITON_CACHE_DIR
+              value: /tmp/.cache/triton
           resources:
             limits:
               nvidia.com/gpu: 1
           volumeMounts:
+            - name: tmp
```

Comment:
we don't need /tmp mount emptyDir , just shm is enough ?

Reply:
that /tmp is not related to shared memory issue